catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -94,50 +79,61 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
     [images]
   );
 
-  const imageRenderer = useCallback(galleryImage => {
-    const photo: GalleryImageProps = galleryImage.photo;
+  const imageRenderer: React.FC<RenderPhotoProps<GalleryImageProps>> = ({
+    photo,
+    layout,
+    wrapperStyle,
+  }) => {
     const rgbColor = hexToRgb(photo.averageColor || '');
-
     return (
-      <ImageContainer key={galleryImage.key} role="listitem">
-        <ImageCard
-          id={photo.id}
-          workId={photo.source.id}
-          image={{
-            contentUrl: photo.src,
-            width: photo.width - imageMargin * 2,
-            height: photo.height,
-            alt: photo.source.title,
+      <div style={wrapperStyle}>
+        <div
+          style={{
+            display: 'block',
+            position: 'relative',
+            width: '100%',
+            height: '100%',
```

Comment:
Could we move this into a styled component?

Reply:
done